bin/update

```diff
@@ -7,7 +7,7 @@ if [[ $# -ne 0 ]]; then
   exit 1
 fi
 
-go_version=go1.13
+go_version=go1.14
 xtools_version=86caa796c7ab0e32b908c5ebd5748faedd06c1da
```

Comment:
where do we usually point xtools_version ...

Reply:
Whatever's current when we update the repo 🤷